### DISCUSSION TITLE
argocd webhooks with github enterprise

### DISCUSSION THREAD

**Discussion by spacebuoy29** [argoproj/argo-cd#9059] [Q&A]
How to use argocd webhooks with github enterprise?
No clear docs available.

**Comment by ishitasequeira** [comment]
I don't think we have any specific documentation for Github Enterprise, but it should be similar workflow as for Github. You can find the GitHub webhook configuration at https://argo-cd.readthedocs.io/en/stable/operator-manual/webhook

**Reply by spacebuoy29** [reply]
yes, I just want to know if anyone has integrated and share their feedback on same.
